.github/workflows/main.yml

```diff
@@ -27,17 +27,9 @@ jobs:
   build-example:
     name: "Build Example App"
     runs-on: macos-latest
-    strategy:
-      matrix:
-        xcode:
-        - '12.4' # Swift 5.3
-        - '12.5.1' # Swift 5.4
-        - '13.2' # Swift 5.5
```

Comment:
I don't think we need to verify that the Example app compiles on multiple Swift versions, since it's not an artifact that we ship